service/service.go

```diff
@@ -492,6 +484,18 @@ func (s *Service) winsHandler(from core.ID, topic string, msg []byte) ([]byte, e
 		return nil, fmt.Errorf("unmarshaling message: %v", err)
 	}
 
+	bid, err := s.store.GetBid(auction.BidID(win.BidId))
+	if err != nil {
+		log.Errorf("error getting bid, assuming bytes limit is not hit: %v", err)
+	} else {
+		// request for some quota, which may be used or gets expired if lossing
+		// the auction.
+		granted := s.bytesLimiter.Request(win.AuctionId, bid.DealSize, BidsExpiration)
+		if !granted {
+			return nil, errors.New("actively reject to avoid hitting running bytes limit")
+		}
```

Comment:
To double-check: So the auctioneer won't receive an ACK, and would move on to the next candidate for winner?

Reply:
Correct. Let me make this error message a constant and have auctioneer skip such errors silently, so no extra protobuf field is required.